app/src/examples/RuntimeTests/ReanimatedRuntimeTestsRunner/RuntimeTestsRunner.tsx

```diff
@@ -1,23 +1,21 @@
 import { View, Button, StyleSheet } from 'react-native';
-import React, { ReactNode, useEffect, useState } from 'react';
+import React, { ReactNode, useEffect, useRef, useState } from 'react';
 import { runTests, configure } from './RuntimeTestsApi';
-import { LockObject } from './types';
-
-let renderLock: LockObject = { lock: false };
 
 export default function RuntimeTestsRunner() {
+  const isRenderLocked = useRef<boolean>(false);
```

Comment:
cc @Latropos

How to properly define this lock inside this component?

Reply:
Isn't it outdated already?